### ISSUE TITLE
Age Ristriction doesnt get set properly

### ISSUE THREAD

**Issue by Anti-ctrl** [jellyfin/jellyfin#5966]
If I set a Profile in Jellyfin to be age restricted, like FSK12, then Jellyfin doesnt properly get it. The Metadata shows Just 12, instead of FSK12, which makes it not possible for Jellyfin to check if the Movie/Show should be streamable, as you can only set "FSK-12/DE-12" in the Options.

**System (please complete the following information):**
 - OS: Ubuntu
 - Virtualization: KVM/Proxmox
 - Clients: all
 - Browser: Chrome
 - Jellyfin Version: 10.7.2
 - Playback: all 
 - Installed Plugins: AniDB, AudioDB, Bookshelf, KodiSyncQueue, MusicBrainz, OMDb, Playback Reporting, TheTVDB
 - Reverse Proxy: nginx
 - Base URL: [e.g. none, yes: /example]
 - Networking: [e.g. Host, Bridge/NAT]
 - Storage: local SMB

**To Reproduce**

1. Go to Profiles Tab
2. Click on the Profile and then on the Age restriction thing (idk the English phrase, sorry)
3. Select any random Rating
4. See the Show/Movie Anyway, even tho its rated higher than the actual restriction you set.

**Expected behavior**
Jellyfin should not show, or Play the Movie/Show, or let me choose an Option where i can set "Age:16" in the Restrictions, instead of only "FSK-16/DE-16"

**Logs**

**Screenshots**
![JF](https://user-images.githubusercontent.com/68434734/117069981-756ad000-ad2d-11eb-9f18-7336653b9c3e.PNG)
![JF](https://user-images.githubusercontent.com/68434734/117069991-77349380-ad2d-11eb-92bf-fe9d1f4da33a.PNG)

**Additional context**
It would be great to have the Option to set every Show/Movie thats imported with an Age rating, to automatically set it to the right Age for you Country. E.G if its Imported with "Age:16", set it automatically to "FSK-16/DE-16" if thats the Rating you choose.

**Comment by Ullmie02** [comment]
What metadata provider are you using for the movie that has 16 set as the age restriction?

**Comment by Anti-ctrl** [comment]
> What metadata provider are you using for the movie that has 16 set as the age restriction?

Mostly i have that Problem with TV-Shows. I use TheTVDB. TheMovieDb, The Open Movie Database or AniDB in that exact order.
The Show which lead to that knowledge was Death Note, but i saw it on other too, after that. They just get a "16" instead of "FSK-16", like (most) movies (all movies of which I checked at random).

**Comment by Ullmie02** [comment]
Do you habe nfo files which may have a "mpaa" attribute? These files overwrite the providers like tvdb and tmdb.
Using only tmdb without an nfo file returns FSK-16 in my testing

**Comment by Anti-ctrl** [comment]
> Do you habe nfo files which may have a "mpaa" attribute? These files overwrite the providers like tvdb and tmdb.

Nope, no NFOs in the directories. If I dont manually overwrite the age, then this is what i will get:
![JF](https://user-images.githubusercontent.com/68434734/117584654-7b3a2a00-b10e-11eb-9072-4599b4dfac2b.PNG)

**Comment by Ullmie02** [comment]
Okay, was just wondering how you got 16 set. The only way I got it to 16 was with a NFO file. I will check again if there are any other possibilities to get 16 set as the age restriction

**Comment by Ullmie02** [comment]
I can now consistently reproduce this with TV Shows but not movies. Could you please send a screenshot of the edit library dialog and a movie name that has a numeric value set as the age restriction like in the screenshots above?

**Comment by Anti-ctrl** [comment]
> Could you please send a screenshot of the edit library dialog and a movie name that has a numeric value set as the age restriction like in the screenshots above?

No i cant. I imagined that it would happen for Movies too and i was wrong about that and didnt checked it in the first place. Shame on me for that!
It really seems to be a TV-Show Problem, as all of the Movies i checked have the Age properly set. Even those, where I didnt fumbled around with the Metadata.
